About editing generated documentation (#575)
MicrosoftDocs/Contribute: Contribute/index.md

   ![Location of the Edit link](./media/index/edit-article.png)

>    If the **Edit** button is not present, it means the content is not open to public contributions.  Some pages are generated, e.g. from inline documentation in code, and must be edited in the project they belong to.
> 

Reviewer: ```suggestion
   If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated, (for example, from inline documentation in code) and must be edited in the project they belong to.

```

>    If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated, (for example, from inline documentation in code) and must be edited in the project they belong to.

Reviewer: ```suggestion
   If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated (for example, from inline documentation in code) and must be edited in the project they belong to.
```
